etc/org.opencastproject.speechtotext.impl.SpeechToTextServiceImpl.cfg

```diff
@@ -1,3 +1,7 @@
 # Job load for a speech-to-text job
 # Default: 0.8
 #job.load.speechtotext=0.8
+
+# Select STT engine.
+# Available engines: vosk, whisper | Default: vosk
+SpeechToTextEngine.target=(engineType=vosk)
```

Comment:
The config key is case-sensitive. Maybe you want to replace the reference name with something just lowercase, like all our config keys.